packages/app-page-builder/src/editor/components/IconPicker.tsx

```diff
@@ -99,17 +146,23 @@ const IconPicker = ({ value, onChange }) => {
                 if (!item) {
                     return null;
                 }
-
+                const isSelectedIcon =
+                    item.id[0] === selectedIconPrefix && item.id[1] === selectedIconName;
                 return (
                     <div
                         key={key}
                         style={style}
-                        className={gridItem}
+                        className={gridItem + (isSelectedIcon ? ` ${gridItemSelected}` : "")}
```

Comment:
For consistency please use `classNames` function here. For example, https://github.com/webiny/webiny-js/blob/master/packages/app-admin/src/plugins/Menu/Navigation/components/Menu.tsx#L60